src/Workspaces/Core/Portable/SolutionCrawler/InvocationReasons_Constants.cs

```diff
@@ -9,76 +9,76 @@ namespace Microsoft.CodeAnalysis.SolutionCrawler
     internal partial struct InvocationReasons
     {
         public static readonly InvocationReasons DocumentAdded =
-            new InvocationReasons(
+            new(
```

Comment:
Could consider removing the extra newlines here?

Reply:
This was an automated refactoring. It would be difficult to modify the code fix rules to unwrap the call only when the user wants it. Would be interested in ideas for obvious cases though.